pandas/plotting/_core.py

```diff
@@ -2613,26 +2618,74 @@ def hist(self, bins=10, **kwds):
 
     def kde(self, bw_method=None, ind=None, **kwds):
         """
-        Kernel Density Estimate plot
+        Kernel Density Estimate plot using Gaussian kernels.
```

Comment:
The short summary should start with an infinitive verb. I would change it to
```
Draw Kernel Density Estimate plot using Gaussian kernels.
```

Reply:
I think for plot functions it is fine to just state the plot type (the rules are to give direction, but sometimes there can be reason to deviate, so in this case the question is if "Draw .." makes it more informative)

Reply:
I agree in that adding `Draw` doesn't add much information, but I would still add a prefix since most (all?) of the doc strings created in the Doc Sprint will be written with these rules in mind. Also, the doc string proposed in [PR20113](https://github.com/pandas-dev/pandas/pull/20113/) for the `hist` function includes `Draw` as well.

On the other hand, `Generate` might be a better fit than `Draw` because out of a jupyter notebook there is nothing drawn ..